scripts/install-jstz-cli.sh

```diff
@@ -72,22 +70,30 @@ EOF
 
     echo "Configuring \`jstz\` alias..."
 
-    shell=$(basename "$SHELL")
-    case "$shell" in
-        "bash")
-            shellrc="$HOME/.bashrc"
-            ;;
-        "zsh")
-            shellrc="$HOME/.zshrc"
-            ;;
-        *)
-            cat 1>&2 << EOF
-Unsupported shell: $shell. 
-Please manually add the alias to your shell's configuration file.
-    $shell_alias
-EOF
-            exit 1
-    esac
+    current_shell=$(ps -p $$ -ocomm=)
```

Comment:
Wouldn't it be cleaner to have:
```suggestion
    current_shell=$(basename "$0")
```

Reply:
$0 points to install-jstz-cli.sh

Reply:
When you source? or run as a script?

Reply:
with source it results in jstz_configure